apps/sensei/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3.7"
+
+services:
+  web:
+    image: getumbrel/sensei:v0.0.2
```

Comment:
You need to build your own Docker image and push it to a repository (usually Docker Hub). I can add an auto build GitHub Action in the sensei repo if you want. Or you can use GitHub images repo.